cypher/cypher-docs/src/docs/dev/query-use.adoc

```diff
@@ -82,32 +82,38 @@ MATCH (n) RETURN n
 ----
 
 
-[role=fabric]
-[[query-use-examples-query-fabric-graph-by-name]]
-=== Query a Fabric graph by name
+[[query-use-examples-query-composite-database-constituent-graph]]
+=== Query a composite database constituent graph
 
-In this example we assume that we have configured a Fabric database called `exampleFabricSetup`.
-The graph that we wish to query is named `exampleDatabaseName`:
+In this example it is assumed that the DBMS contains a composite database named `myComposite`, containing an alias named `myConstituent`:
 
 .Query.
 [source, cypher]
 ----
-USE exampleFabricSetup.exampleDatabaseName
+USE myComposite.myConstituent
 MATCH (n) RETURN n
 ----
 
 
-[role=fabric]
-[[query-use-examples-query-graph-by-graph-id]]
-=== Query a Fabric graph by graph ID
+[[query-use-examples-query-composite-database-constituent-graph-dynamically]]
+=== Query a composite database constituent graph dynamically
 
-This examples continues with a Fabric database called `exampleFabricSetup`.
+The built-in function `graph.byName()` can be used in the `USE` clause to resolve a constituent graph from a string value containing the qualified name of a constituent.
```

Comment:
```suggestion
The built-in function `graph.byName()` can be used in the `USE` clause to resolve a constituent graph from a string value including the qualified name of a constituent.
```

Reply:
It seems you're not very fond of the words describing containment, like "contain" and "in", and I'm fine with that :smile: But in this case "include" does not work.

Reply:
It's just that it doesn't read very naturally in English, but it's still not wrong 😊